sentry-android-core/src/main/java/io/sentry/android/core/performance/ActivityLifecycleTimeSpan.java

```diff
@@ -0,0 +1,29 @@
+package io.sentry.android.core.performance;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public class ActivityLifecycleTimeSpan implements Comparable<ActivityLifecycleTimeSpan> {
+  private final @NotNull TimeSpan onCreate = new TimeSpan();
+  private final @NotNull TimeSpan onStart = new TimeSpan();
+
+  public final @NotNull TimeSpan getOnCreate() {
+    return onCreate;
+  }
+
+  public final @NotNull TimeSpan getOnStart() {
+    return onStart;
+  }
+
+  @Override
+  public int compareTo(ActivityLifecycleTimeSpan o) {
+    final int onCreateDiff =
+        Long.compare(onCreate.getStartUptimeMs(), o.onCreate.getStartUptimeMs());
+    if (onCreateDiff == 0) {
+      return Long.compare(onStart.getStartUptimeMs(), o.onStart.getStartUptimeMs());
```

Comment:
guess it's `getProjectedStopTimestampMs`?

Reply:
I thinks it's fine this way, if `onCreate` is the same, then use `onStart`. I wouldn't add any end timestamps for sorting here.